journalist_gui/requirements.txt

```diff
@@ -0,0 +1,2 @@
+PyQt5
```

Comment:
Because these packages are imported on admin workstations, I suggest we generate this file like we do for other requirements and pin versions/hashes.

Reply:
So something I should have stated explicitly is that this requirements file is only used in the development environment. One constraint of electing to use Python 3 without a virtual env is that we are only using Python packages that are present in Tails already. I thought this would be fine to do as we don't _really need_ any specialized packages in this simple client. If we do ever want to include additional packages, we would need to replicate a similar setup as `securedrop-admin`, with checksums.

Reply:
Added a note in the developer documentation about this 
 in b3fa1829aafd08dfbf63810c7e6853bd44728b67